numba_cuda/numba/cuda/api.py

```diff
@@ -95,6 +95,14 @@ def is_float16_supported():
     return True
 
 
+def is_bfloat16_supported():
+    """Whether bfloat16 are supported.
+
+    bfloat16 are only supported on devices with compute capability >= 8.0
+    """
+    return current_context().device.supports_bfloat16
```

Comment:
Do we want to check for CUDA 12 here as well?

Reply:
Does it look a bit strange that `Device.supports_bfloat16` and `api.is_bfloat16_supported` has different semantics?